database/mockData.js

```diff
@@ -0,0 +1,46 @@
+const {dropDB, save} = require('../database/index');
+const bodyParser = require('body-parser');
+
+
+var mockData = [];
+
+var randomColor = function() {
+  var letters = '0123456789ABCDEF';
+  var color = '#';
+  for (var i = 0; i < 6; i++) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+};
+
+var colorOptions = function() {
+  var list = [];
+  var randomAmount = Math.floor(Math.random() * 6 + 1);
+  for (var i = 0; i < randomAmount; i++) {
+    list.push(randomColor());
+  }
+  return list;
+};
+
+
+var randomPrice = function() {
+  var dollars = Math.floor(Math.random() * 100 + 50);
+  return dollars + 0.99;
+};
+
+for (var i = 0; i < 101; i++) {
```

Comment:
Note that this does load in 101 helmets.. I think that's probably fine, but it isn't exactly 100.